powerlevel9k.zsh-theme

```diff
@@ -1623,6 +1623,9 @@ $(print_icon 'MULTILINE_LAST_PROMPT_PREFIX')'
 NEWLINE='
 '
   [[ $POWERLEVEL9K_PROMPT_ADD_NEWLINE == true ]] && PROMPT="$NEWLINE$PROMPT"
+
+  # Allow iTerm integration to work
+  [[ $ITERM_SHELL_INTEGRATION_INSTALLED == "Yes" ]] && PROMPT="%{$(iterm2_prompt_mark)%}$PROMPT"
```

Comment:
Does `ITERM_SHELL_INTEGRATION_INSTALLED` get set automatically by iTerm2? And it gets set to the string `"Yes"`, not `true`? I'm asking because I don't know, hah.

Same questions for `iterm2_prompt_mark`.

Reply:
`ITERM_SHELL_INTEGRATION` gets set to `Yes` in the iTerm shell integration script, which you have to download and source separately after installing iTerm if you want that functionality. The instructions are on their website.